src/api/app/models/package.rb

```diff
@@ -929,8 +929,6 @@ def buildresult(prj = project, show_all = false)
   end
 
   def jobhistory_list(project, repository, arch, options = {})
-    options[:package] = name if options[:package].blank?
-
```

Comment:
So you could drop the whole options and make this a parameter or keyword parameter.

Reply:
that is what I did, didn't I?

Reply:
I was reviewing per commits and didn't see that at that time:wink: